src/View/Helper/ImageHelper.php

```diff
@@ -40,6 +40,17 @@ public function display($image, $version = null, $options = array()) {
 		return $this->fallbackImage($options, $image, $version);
 	}
 
+	protected function _getHash($version, $image) {
```

Comment:
I'm wondering if hash shouldn't be resolved in a listener instead of client classes. Currently we have at least two places where this code occurs. And it should be copied to every class which triggers `ImageVersion.getVersions` event.